hydrate.py

```diff
@@ -34,6 +36,36 @@ def get_deployments_for_all_namespaces():
 
 
 if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description='Generate a component.yaml file for your cluster.')
+    parser.add_argument(
+        'run',
+        help='Generate component.yaml for current configuration.')
+    parser.add_argument(
+        '-n', '--name',
+        action='store',
+        default='hydrated-cluster',
+        help='Specify the name of the main component.')

```

Comment:
Another way of putting the help can be "Name of the main component (default: hydrated-cluster)"

Reply:
Added